src/terrain/terrain.js

```diff
@@ -740,6 +741,15 @@ export class Terrain extends Elevation {
         return {efficiency: (1.0 - uncacheableLayerCount / drapedLayerCount) * 100.0, firstUndrapedLayer};
     }
 
+    getMinElevationBelowMSL(): number {
+        let min = 0;
```

Comment:
Nice. I think this is a better initialization than +INFINITY, since when things were initially loading, I found that infinity percolated through to the camera adjustment.